### PR TITLE
Updated geometry for Presidio Heights neighbourhood

### LINKED ISSUE #127: Presidio Heights (SF) has delusions of grandeur
It's like when Alberta said they should build a navy....

https://whosonfirst.mapzen.com/spelunker/id/85865911/

screen shot 2015-10-26 at 08 10 03

### PR DESCRIPTION
**Pull request by stepps00**
fixes #127

### CODE REVIEW

**Comment by stepps00** [comment]
@nvkelso This pull request now contains edits to the North Beach and Presidio Heights neighborhoods, as well as a new record for Treasure Island. Requesting that this PR is merged to Master.

File: data/823/321/587/823321587.geojson [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    "lbl:longitude":-122.37108,
+    "mz:is_funky":0,
+    "mz:is_hard_boundary":0,
+    "mz:is_landuse_aoi":1,
+    "mz:is_official":0,
+    "mz:max_zoom":18,
+    "mz:min_zoom":15,
+    "mz:tier_locality":7,
+    "src:geom":"mapzen",
+    "src:geom_alt":[
```
Comment (nvkelso) [inline data/823/321/587/823321587.geojson]: There is no `geom_alt` for this feature, remove lines 22 to 24.

**Comment by nvkelso** [comment]
The Financial District portion is covered in a separate PR: https://github.com/whosonfirst/whosonfirst-data/pull/283.

**Comment by nvkelso** [comment]
Modulo alt geometry discussion in https://github.com/whosonfirst/whosonfirst-data/pull/283, 👍.

**Comment by stepps00** [comment]
@nvkelso - 1150db3 now includes the following:

* Removed `src:geom` from the new Treasure Island record
* Updated `src` fields, created a new zetashapes alt geojson, and Exportified all records for North Beach
* Updated `src` fields, created a new zetashapes alt geojson, and Exportified all records for Presidio Heights

**Comment by nvkelso** [comment]
Almost there! But I'm seeing the new `mapzen` geom in the `zetashapes` alt-geom file. Let's go through how to dig out the old zetashape geom in the git history together on Thursday.

**Comment by stepps00** [comment]
Dug out the old zetashape geometries in the git history and replaced the `mapzen` geometries with the new new `zetashape` geometries in the alt-geom files.

The most recent [commit](5508bca) essentially replaced the alt-geom geometries with the geometries that were originally associated to `src: zetashapes`. All geometries should now associate to the appropriate source.

**Comment by nvkelso** [comment]
@stepps00 Looks like the geometry in **85865911-alt-zetashapes.geojson** is corrupted? I can't preview it on Github.com, and won't load in GeoJSON.io, and fails jsonlint.com.

**Comment by nvkelso** [comment]
(Everything else looks good, fix the above and send to Aaron.)

**Comment by stepps00** [comment]
[Most recent commit](f38ed41) takes care of the invalid geojson.
The spacing/indent issue was due to the geojson geometry `type` being listed as a `polygon`, rather than a `multipolygon`.

**Comment by stepps00** [comment]
Over to Aaron for validation.

**Comment by thisisaaronland** [comment]
👍

**Pull request merged**